app/javascript/onboarding/components/PersonalInfoForm.jsx

```diff
@@ -66,10 +66,10 @@ class PersonalInfoForm extends Component {
       },
       body: JSON.stringify({
         user: {
-          last_onboarding_page,
+          lastOnboardingPage,
           location,
-          employer_name,
-          employment_title,
+          employerName,
```

Comment:
This information doesn't seem to be making it through, this probably needs to go back to snake case as well.